### PR TITLE
Jripke74

### PR DESCRIPTION
**Pull request by jripke74**
add hometown lat long @githubteacher close #3797

### CODE REVIEW

**Comment by brianamarie** [comment]
Hi @jripke74! It looks like the tests are failing because of the name of the file. Make sure the name is `_pins/jripke74.json`; right now it's just `jripke74.json`. Let me know if you need any help!

**Comment by brianamarie** [comment]
Hi @Unix3g! Let's leave this pull request open until @jripke74 merges it. 😄

**Comment by kaungmyatnoe** [comment]
@jripke74 This looks great! 🎉

**Comment by susanli2016** [comment]
@jripke74 This looks good 👍

File: _pins/jripke74.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: jripke74
+latitude: 47.608013
+longitude: -122.335167
+---
```
Comment (jlabuac) [inline _pins/jripke74.json]: Looks good

**Pull request merged**